docs/content/preview/yugabyte-platform/security/enable-encryption-in-transit.md

```diff
@@ -105,9 +105,12 @@ You can also modify TLS settings for an existing universe by navigating to **Uni
 
 For universes created with an on-premise cloud provider, instead of using self-signed certificates, you can use third-party certificates from external CAs. The third-party CA root certificate must be configured in YugabyteDB Anywhere. You have to copy the custom CA root certificate, node certificate, and node key to the appropriate database nodes using the procedure described in [Use custom CA-signed certificates to enable TLS](#use-custom-ca-signed-certificates-to-enable-tls).
 
-The certificates must meet the following criteria:
+The certificates must adhere to the following criteria:
+
+- Be in the `.crt` format and the private key must be in the `.pem` format. 
```

Comment:
A .crt file could be either DER encoded or PEM encoded. We should probably update this text to specify that the certificate and key must be in PEM format rather than specifying the file extension.

Reply:
Is this better?

Be stored in a `.crt` file, with both the certificate and the private key being in the PEM format.